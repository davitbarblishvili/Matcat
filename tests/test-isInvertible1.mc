func main() int {
    matrix a;
    a = [[5,6],[8,8]];
    isInv(a);

    return 0;
}