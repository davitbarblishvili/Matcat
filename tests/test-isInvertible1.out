This matrix is invertible
calculating inverse...
(
[-1.00 0.75 ]
[1.00 -0.62 ]
)